python/ray/util/tracing.py

```diff
@@ -24,14 +24,15 @@
 try:
     from opentelemetry import context, trace
     from opentelemetry.context.context import Context
-    from opentelemetry.util.types import Attributes
+    # from opentelemetry.util.types import Attributes
     from opentelemetry import propagators
     from opentelemetry.trace.propagation.textmap import DictGetter
 except ImportError:
     if os.getenv("RAY_TRACING_ENABLED", "False").lower() in ["true", "1"]:
-        logger.warning(
-            "Install opentelemetry with 'pip install opentelemetry-api' and"
-            "'pip install opentelemetry-sdk' to enable tracing.")
+        raise Exception(
```

Comment:
Please raise a more specific exception type here. I'm not sure what the right one is, `RuntimeError`?

Reply:
not ImportError?

Reply:
That seems good